statistics/handle/update.go

```diff
@@ -162,9 +162,9 @@ func (s *SessionStatsCollector) Update(id int64, delta int64, count int64, colSi
 
 var (
 	// MinLogScanCount is the minimum scan count for a feedback to be logged.
-	MinLogScanCount = int64(1000)
+	MinLogScanCount = int64(1)
 	// MinLogErrorRate is the minimum error rate for a feedback to be logged.
-	MinLogErrorRate = 0.5
+	MinLogErrorRate = 0.0
```

Comment:
Should we change them back to 1000 and 0.5?

Reply:
Yeah, it's used for testing